xmtp_mls/src/storage/encrypted_store/mod.rs

```diff
@@ -146,6 +146,8 @@ impl EncryptedMessageStore {
             conn.batch_execute(&format!("PRAGMA key = \"x'{}'\";", hex::encode(key)))?;
         }
 
+        conn.batch_execute("PRAGMA busy_timeout = 5000;")?;
```

Comment:
Are we seeing busy errors that take more than 5s?

Reply:
Tbh, this change did nothing. We aren't seeing busy errors at all, since the DB locks are a different error code altogether.